recipes/example/meta.yaml

```diff
@@ -45,6 +45,7 @@ about:
   # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
   # See https://opensource.org/licenses/alphabetical
   license: MIT
+  license_family: MIT
```

Comment:
Sorry, I misunderstood https://github.com/conda-forge/staged-recipes/pull/1838#discussion_r85647035 . I included a comment, but I'm still not sure if it's correct. Is it always optional or just if it's identical to `license`?